Makefile

```diff
@@ -15,3 +15,17 @@ export Q
 # ---------------------------------------------------------------------------
 htmldocs:
 	$(Q)$(MAKE) -C doc htmldocs
+
```

Comment:
I think the intent of this Makefile is only as a convenience for the CI process generating the documentation.   I'll let the dev team speak up about this.